_sub/monitoring/helm-exporter/vars.tf

```diff
@@ -0,0 +1,48 @@
+variable "cluster_name" {
+  type = string
+}
+
+variable "deploy_name" {
+  type        = string
+  description = "Unique identifier of the deployment, only needs override if deploying multiple instances"
+  default     = "helm-exporter"
+}
+
+variable "namespace" {
+  type        = string
+  description = "The namespace in which to deploy Helm resources"
```

Comment:
Since it is intended that this one is installed in the monitoring namespace, then please add:

`default = "monitoring"`

Reply:
Thanks, added!